beta-src/src/components/ui/main-screen/buttons/OpenModalButton.tsx

```diff
@@ -109,10 +108,6 @@ const OpenModalButton: FunctionComponent<BottomRightProps> = function ({
     setIsComponentVisible(!isComponentVisible);
   };
 
-  useEffect(() => {
-    setIsComponentVisible(true);
```

Comment:
with this change, the press is not open when the user enters the game for the first time, are you sure you want that?